src/main/java/com/knu/buga1chuk/data_structure/QueueDemo.java

```diff
@@ -10,7 +16,8 @@ public static void main(String[] args) {
         }
 
         int removedFirstEl = queue.poll();
```

Comment:
What does it mean `el`?
Always write full word

Reply:
Fix